OJP/OJP_Requests.xsd

```diff
@@ -35,7 +43,13 @@
 	<xs:annotation>
 		<xs:documentation>========================================== Location Information Service ==========================================</xs:documentation>
 	</xs:annotation>
-	<xs:element name="OJPLocationInformationRequest" type="OJPLocationInformationRequestStructure" substitutionGroup="siri:AbstractFunctionalServiceRequest"/>
+	<xs:element name="OJPLocationInformationRequest" type="OJPLocationInformationRequestStructure" substitutionGroup="siri:AbstractFunctionalServiceRequest">
+		<xs:annotation>
+			<xs:documentation>The Location Information service comprises the following four base functions:
+matching text input against possible origin and destination locations, retrieval of all location objects (bunch delivery), geographical context service that provides location objects within a bounding box, reverse address resolution service that delivers the nearest address for a given coordinate.
+By means of abstraction these functions are assembled within one single service – the Location Information service. This way, even more possible applications have arisen, such as: finding the nearest stops/stations for a given coordinate, matching text input against the names of locations near a given coordinate. This service implements PI QR Location Request from TM 6.</xs:documentation>
```

Comment:
QR?

TM 6 -> TM6? (consistency)

Reply:
This are all packages from TM  
![image](https://github.com/VDVde/OJP/assets/24227470/47ab218b-f706-42cc-b8dc-a19df3e252b5)